orographic_precipitation/fastscape_ext.py

```diff
@@ -135,6 +136,6 @@ def _discharge(self):
 
 
 precip_model = basic_model.update_processes({
-    'orographic': OrographicPrecipitation,
+    'precipitation': OrographicPrecipitation,
```

Comment:
nit: we've chosen "orographic" as process name in this model because when creating a simulation setup "orographic__precip_base" looks better than "precipitation__precip_base" as input name (same for the other input or output variable in this process).

Reply:
Sure no problem, I put it back to orographic.